trajopt_optimizers/trajopt_optimizers_test_suite/CMakeLists.txt

```diff
@@ -0,0 +1,97 @@
+cmake_minimum_required(VERSION 3.5.0)
+project(trajopt_optimizers_test_suite)
+
+find_package(console_bridge REQUIRED)
+# find_package(Pagmo REQUIRED)
+find_package(OsqpEigen REQUIRED)
+find_package(trajopt_ifopt REQUIRED)
+find_package(trajopt_utils REQUIRED)
+# find_package(trajopt_pagmo REQUIRED)
+find_package(trajopt_sqp REQUIRED)
+find_package(ifopt REQUIRED)
+find_package(Eigen3 REQUIRED)
+find_package(GTest REQUIRED)
+
+###########
+## Build ##
+###########
+
+add_library(${PROJECT_NAME} SHARED
```

Comment:
Throughout, remove the commented code.

I believe this can be an interface library once you move the tests from the src files into the test directory. See [HERE](https://github.com/ros-industrial-consortium/tesseract/blob/92722baaa2cfc152d6a2bd79ff0485dfbb780cde/tesseract/tesseract_kinematics/CMakeLists.txt) for example.

Reply:
Lets get rid of run_tests.cpp and make this an interface target